omero/users/whatsnew.rst

```diff
@@ -1,35 +1,23 @@
-What's new for OMERO 5.3 for users
+What's new for OMERO 5.4 for users
 ==================================
 
-Updates and new features for OMERO 5.3 include:
+Updates and new features for OMERO 5.4 include:
 
-- ROI Folders in OMERO.insight and the |CLI| which allow you to organize your
-  ROIs into a hierarchical tree structure of folders so they can be sorted
-  e.g. by phenotypes or ontologies. Support for these is still upcoming for
-  OMERO.web. There are also some permissions issues in OMERO.insight meaning
-  that at present, even in groups where the permissions should allow this, you
-  cannot add ROIs created by other people to Folders. You can perform this
-  action using the CLI however, and OMERO.insight will then display the Folder
-  contents correctly
-- support for lookup tables (LUT) and reverse/inverted intensity rendering
-- support for histograms
-- the clients UI for viewing Screen Plate Well data has been redesigned to
-  show the spatial layout of fields within each Well
-- Wells can now be annotated in the clients
-- Wells can also be searched by annotations, and a new script allows you to
-  migrate existing annotations from Images to Wells (see
-  :help:`utility scripts <scripts.html#utility>`)
-- the new 'Open With...' functionality allows images to be opened with
-  OMERO.figure or other plugins directly from OMERO.web (coming soon is a new
-  web-based full image viewer - OMERO.iviewer)
-- ability to filter by ratings in OMERO.web
-- updated Bio-Formats bringing improved file format support including support
-  for JPEG-XR compressed CZI data
+- Your system administrator can now create
+  :doc:`/sysadmins/admins-with-restricted-privileges` to allow facility
+  managers, image analysts etc. to organize users and data in OMERO
+  without having to be granted full administrator privileges. This is a way to
+  cater for the needs for some trusted users to act on behalf of all the OMERO
+  users in a facility, with access to all the groups and data in OMERO. Full administrators and administrators with restricted privileges can now create Project/Dataset/Screen on behalf of other users in OMERO.web or transfer all the data of a given user using the :program:`chown` command.
 
-See the :help:`User help website <>` for information on how to incorporate
-these new features into your current workflows. There is also a
-`short demo movie <https://downloads.openmicroscopy.org/movies/omero-5-3/mov/OMERO.web-5.3.0.mov>`_ for the new features of OMERO.web.
+- Improvements to OMERO.web including the display of plates and wells, the
+  synchronization of controls enabling copy/paste of rendering settings, the display of 32-bit and float images, the layout of the left-hand panel and the
+  ability to export images in plate as OME-TIFF.
```

Comment:
I don't remember what the 'synchronization of controls' was. The left-hand panel looks almost identical (DOM & css cleanup). The fix for float images is only noticeable for images with small range.

I would say something like: "Other improvements to OMERO.web include the display of fields within Wells, handling of float images with small dynamic range and the ability to export images in plate as OME-TIFF as well as a number of minor bug fixes." ?